core/commands/public_keys_command.rb

```diff
@@ -41,6 +36,21 @@ def show_help
 
   private
 
+  # Сopies the ssh key to available nodes.
+  def copy_key_to_node
+    available_nodes = @network_config.store_network_config
```

Comment:
You must use the IP addresses that are defined in the network settings. Please remove dependency upon the NetworkConfig class. It only supports Vagrant.